guides/v2.1/javascript-dev-guide/widgets/widget_modal.md

```diff
@@ -176,6 +176,32 @@ The modal widget is subscribed to the following events:
 <li><a href="#modal_opened">opened</a></li>
 </ul>
 
+These can be listened to a couple of different ways, the primary being listening to it with jQuery's [`on`](http://api.jquery.com/on/) function, or creating a property with the name of the event that you want to listen to, like so:
+
+<pre>
+$('#modal_content').modal({
+    &lt;option1&gt;: &lt;value1&gt;,
+    &lt;option2&gt;: &lt;value2&gt;,
+    ...
+});
+$( "#modal_content" ).on( "close", function() {
```

Comment:
Event must be `modalclosed`, not simply 'closed'

Reply:
That's interesting, how do you determine how to convert the two?

For example, the documentation says the modal listens and fires the 'close' event, how does jquery convert close -> modalclosed? The property is also close and it seems to work for me.

Am i missing something?

Reply:
You can read about it here https://api.jqueryui.com/jquery.widget/.
> widgetEventPrefix: The prefix prepended to the name of events fired from this widget. For example the widgetEventPrefix of the draggable widget is "drag", therefore when a draggable is created, the name of the event fired is "dragcreate". **By default the widgetEventPrefix of a widget is its name.**

Reply:
Ah, I see, i mis-remembered what the property was called, so this documentation needs to be changed a bit, my apologies!

https://github.com/magento/magento2/blob/2.2-develop/app/code/Magento/Ui/view/base/web/js/modal/modal.js#L327

Looking at this it makes sense, sorry for all the questions!